### DISCUSSION TITLE
error in settings.json

### DISCUSSION THREAD

**Discussion by namobalsacc** [Pilot1782/bad_copeheimer#101] [Q&A]
![rederror](https://user-images.githubusercontent.com/122157271/211150528-db0ce67e-1338-42cd-8f3c-975cfb4ee2f0.PNG)

is all red ive been tryiong to fix it for like 2hours alr please help :3

**Comment by CaptainRexPL** [comment]
> ![rederror](https://user-images.githubusercontent.com/122157271/211150528-db0ce67e-1338-42cd-8f3c-975cfb4ee2f0.PNG)
> 
> is all red ive been tryiong to fix it for like 2hours alr please help :3

Add double / in the directory so instead of
"D:\copenheimer-main\bad_copenheimer-main\"
it should be:
"D:\\copenheimer-main\\bad_copenheimer-main\\"

**Reply by Pilot1782** [reply]
It should be `D:\\copenheimer-main\\bad_copenheimer-main\\` bc the last backslash is acting as an escape character.

**Reply by CaptainRexPL** [reply]
Oh, yes. I forgot to add the backslashes in the correct places here after I copied it.

**Reply by namobalsacc** [reply]
![image](https://user-images.githubusercontent.com/122157271/211166341-f7c1f2e4-d5ea-4a5a-85fc-79cf943136da.png)
why when i do the python3 setup.py thing it resets settings.json to error

**Reply by Pilot1782** [reply]
I haven't touched the setup script in a long time as the newer scripts (mongoBot and scanCore) don't need it, pretty soon I'll be fully transitioning over to them and removing the older scripts (dis-bot, setup, settings, .env, etc). As for right now just delete the last \ in the path.

**Reply by namobalsacc** [reply]
still same when python3 setup.py it reverts settings .json and gives red errors

**Reply by namobalsacc** [reply]
so how can i get it to run properly? do you have discord?

**Comment by Pilot1782** [comment]
[Heres an online formatter](https://elmah.io/tools/json-formatter/) you might find useful.

**Reply by namobalsacc** [reply]
why when i do the python3 setup.py thing it resets settings.json to error
![image](https://user-images.githubusercontent.com/122157271/211166470-0d3e56df-f028-4f5c-b0f4-521072c191d3.png)

**Reply by namobalsacc** [reply]
is there a bot thats already been made that i could just add to a server?

**Reply by CaptainRexPL** [reply]
I don't think so. The easiest way to host the bot is through replit. And as mentioned above- discord.pyw won't fully work and you should just fill the mongoBot with required data and run this instead of trying to fix/host the old version.

**Reply by namobalsacc** [reply]
can i add u on discord so its easier to communicate?

**Reply by namobalsacc** [reply]
> I don't think so. The easiest way to host the bot is through replit. And as mentioned above- discord.pyw won't fully work and you should just fill the mongoBot with required data and run this instead of trying to fix/host the old version.

wait if its on replit then whats the bot directory

**Reply by CaptainRexPL** [reply]
In mongobot.pyw from dev-builds the bot doesn't depend on any other files so settings file does not matter.

**Reply by namobalsacc** [reply]
so how would i run python3 setup.py in replit

**Reply by CaptainRexPL** [reply]
You don't need any other file that mongobot.pyw. You just have to install the required libraries and that's it. Setup.py, dis-bot.pyw or any other file doesn't matter if you use mongobot.

**Reply by namobalsacc** [reply]
![image](https://user-images.githubusercontent.com/122157271/211167277-486892df-2004-4862-b93e-7edea52bafd9.png)
add me on discord Namo#0002

**Reply by CaptainRexPL** [reply]
Just to verify if the account I see is correct- you are on The Fifth Column's public discord server, right?

**Reply by namobalsacc** [reply]
yes im in the discord

**Reply by CaptainRexPL** [reply]
Ok, I added you.

**Reply by Pilot1782** [reply]
> ![image](https://user-images.githubusercontent.com/122157271/211167277-486892df-2004-4862-b93e-7edea52bafd9.png)
> 
> add me on discord Namo#0002

Replit doesn't use pip, in the package manager search and add McStatus.
